### ISSUE TITLE
You've found something that doesn't exist

### ISSUE THREAD

**Issue by harikt** [phly/phly-mustache#4]
Hi @weierophinney , 

All the doc pages rendered are giving 404 : http://phly-mustache.readthedocs.org/en/latest/usage.html

There is no such google groups : https://groups.google.com/forum/#!forum/phly-mustache

Also the change log points to old repo PR?

**Comment by weierophinney** [comment]
The URL sounds be http://phly-mustsche.readthedocs.org/en/latest/usage/ - which resolves fine. Where are you seeing the URL you provided?

Also, yes, the changelog refers to the old repository, as that's where the pull requests were made; I renamed the repo (well, created the new repo; I didn't want the old repo links to fail for users of the original package on packagist) _after_ meeting all of them, but before tagging.

I'll investigate the Google group; the name may be with an underscore (I did a sed to replace old mentions of the old repo name).
